lib/config/validation.spec.ts

```diff
@@ -29,25 +30,7 @@ describe('config/validation', () => {
       expect(warnings).toMatchSnapshot();
     });
 
-    it('catches global options in repo config', async () => {
-      const config = {
-        binarySource: 'something',
-        username: 'user',
-      };
-      const { warnings } = await configValidation.validateConfig(false, config);
-      expect(warnings).toHaveLength(2);
-      expect(warnings).toMatchObject([
-        {
-          message: `The "binarySource" option is a global option reserved only for bot's global configuration and cannot be configured within repository config file`,
-        },
-        {
-          message: `The "username" option is a global option reserved only for bot's global configuration and cannot be configured within repository config file`,
-        },
-      ]);
-    });
-
-    // false globals are the options which have names same to the another globalOnly option
-    it('does warn for false globals in repo config', async () => {
+    it('does not warn for false globals in repo config', async () => {
```

Comment:
```suggestion
    it('only warns for actual globals in repo config', async () => {
```

Rewriting the test name to avoid the double negative. Please check if this is correct. :wink: